### ISSUE TITLE
Boutiques descriptor for containers?

### ISSUE THREAD

**Issue by stebo85** [NeuroDesk/neurocontainers#217]

**Comment by stebo85** [comment]
@Vbitz @astewartau 

where are we with this? Is this working reliably? What else needs to be done?

**Comment by astewartau** [comment]
The latest QSMxT recipe that is currently building has a boutiques descriptor that we would like to test. This would be a good test case to confirm the functionality of the boutiques descriptors, as the QSMxT usage is rather complex. Currently, I believe it will just create the descriptor and package it alongside the neurocontainer (@Vbitz ?). I think we are missing a documented way of running these descriptors in connection to Neurodesk.

**Comment by Vbitz** [comment]
Yes currently it will package the descriptor inside the container.

**Comment by Vbitz** [comment]
@stebo85 I'm guessing we need some test or some system to pull these descriptors then some bulk work in pulling existing descriptors into current containers.

**Comment by stebo85** [comment]
yes
